core/src/main/kotlin/com/alecstrong/sql/psi/core/psi/mixins/ColumnNameMixin.kt

```diff
@@ -32,3 +36,18 @@ internal abstract class ColumnNameMixin(
     return AllIcons.Nodes.DataColumn
   }
 }
+
+fun SqlColumnName.getColumnDefOrNull(): SqlColumnDef? {
```

Comment:
I think it's a little more nuanced then this, since columns can come from an alter table statement

Reply:
I'm also pretty sure this is already done elsewhere in code quite a bit (going from column name to column def)

Reply:
Thanks, you are right. Also added a test for alter table.